sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample1_DetectLanguage.md

```diff
@@ -1,6 +1,6 @@
 # Detecting the Language of Text Inputs
 
-This sample demonstrates how to detect the language that one or more text inputs are written in using Azure Text Analytics.  To get started you'll need a Text Analytics endpoint and credentials.  See [README](../README.md) for links and instructions.
+This sample demonstrates how to detect the language that one or more text inputs are written in using Azure Text Analytics.  To get started you'll need a Text Analytics endpoint and credentials.  See [README][README] for links and instructions.
```

Comment:
I'd drop the "using Text Analytics" part. Seems implied and makes the sentence flow better.